### PR TITLE
add addon emmet

### PR DESCRIPTION
**Pull request by ChenZerong**
HTML panel support shorthand [#10](https://github.com/egoist/codepan/issues/10)

### CODE REVIEW

**Comment by egoist** [comment]
Seems it's very big, maybe we can set it up dynamically using `import()`

**Comment by ChenZerong** [comment]
Do you mean module `emmet-codemirror` ?

**Comment by egoist** [comment]
yep

**Comment by ChenZerong** [comment]
right this time?

File: src/utils/create-editor.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ export default function (el, opts = {}) {
   })
 
   if (opts.mode === 'htmlmixed') {
-    emmet.default(editor)
+    emmet(editor)
```
Comment (egoist) [inline src/utils/create-editor.js]: I mean `import('emmet-codemirror').then`

Comment (ChenZerong) [inline src/utils/create-editor.js]: got it...

**Comment by ChenZerong** [comment]
Could you review again? 👀

**Comment by egoist** [comment]
Looking good! But where did you find the usage of `emmet.setup(CodeMirror)`?

**Comment by egoist** [comment]
OK, I got it.

**Comment by ChenZerong** [comment]
I just looked at the addon_emmet source, and then found that the lack of some global variables, and then found the webpack version, and `setup` interface 😄  These days a holiday, I wish you have fun to have fun

**Pull request merged**